src/app/dashboard/student-preview-card/student-preview-card.component.scss

```diff
@@ -0,0 +1,161 @@
+@import '../../../scss/_style-constants.scss';
+
+/* Student Name Styles */
+
+.student-name-wrapper {
+  overflow: hidden;
+  width: 10rem;
+}
+
+.student-gender-text {
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
Since there are only line-height and margin-bottom left, you can think of removing this class and use `mb-0` and` lh-sm` instead. This could also be true for some other classes